src/components/atoms/Burger/index.tsx

```diff
@@ -1,13 +1,12 @@
-import React, { useState } from 'react'
+import React, { useState, ReactComponentElement } from 'react'
 import { StyleBurger, StyleDiv } from "./styles"
 
-
-const BurgerIcon: React.FC<{ open: boolean, setOpen: Function }> = ({ open, setOpen }) => {
+const BurgerIcon: React.FC<{ open: boolean, setOpen: Function }> = (props) => {
     return (
-        <StyleBurger open={open} onClick={() => setOpen(!open)}>
-            <StyleDiv open={open} />
-            <StyleDiv open={open} />
-            <StyleDiv open={open} />
+        <StyleBurger open={props.open} onClick={() => props.setOpen(!props.open)}>
```

Comment:
I don't how to explain this but is safer to use 
`onClick={() => setOpen(oldProps => !oldProps)}`

Might be a good idea if you ask on stackoverflow or reactiflux the difference between:
`onClick={() => setOpen(oldProps => !oldProps)}`
and
`onClick={() => setOpen(!open)}`

Reply:
Like this?

        <StyleBurger open={props.open} onClick={() => props.setOpen(!props.open)}>

Reply:
Like this

`    <StyleBurger open={props.open} onClick={() => setOpen(oldProps => !oldProps)}`

`setOpen` is receiving the previous/old props as a value.